core/components/css-utilities/Display/Display.story.tsx

```diff
@@ -75,28 +69,289 @@ export const display = () => {
           />
         </Card>
       </div>
-      <Heading size="m">Examples</Heading>
-      <Paragraph>Here are some representative examples of these classes:</Paragraph>
-      <br />
-      <div className="d-inline p-2 Utilities-example">d-inline</div>
-      <div className="d-inline p-2 bg-dark Utilities-example Utilities-text--color">d-inline</div>
-      <br />
-      &nbsp;
-      <div className="DocPage-codeBlock pb-5 pt-5 pl-5">
-        {'<div className="d-inline p-2">d-inline</div>'}
-        <br />
-        {'<div className="d-inline p-2">d-inline</div>'}
-        <br />
+
+      {/* Examples section */}
+      <Heading size="m" className="mt-8">
+        Examples
+      </Heading>
+      <Text appearance="default">Here are examples of these display utilities in action:</Text>
+
+      {/* d-inline example */}
+      <div className="mt-8 mb-8">
+        <Heading size="s" className="mb-3">
+          Inline Display
+        </Heading>
+        <div className="mb-4">
+          <Text appearance="default" className="d-inline bg-secondary-lightest border p-4 mr-2 mb-2 align-middle">
+            d-inline
+          </Text>
+          <Text appearance="default" className="d-inline bg-secondary-lightest border p-4 mr-2 mb-2 align-middle">
+            d-inline
+          </Text>
+          <Text appearance="default" className="d-inline bg-secondary-lightest border p-4 mr-2 mb-2 align-middle">
+            d-inline
+          </Text>
+          <Text appearance="default" className="d-inline bg-secondary-lightest border p-4 mr-2 mb-2 align-middle">
+            d-inline
+          </Text>
+          <Text appearance="default" className="d-inline bg-secondary-lightest border p-4 mr-2 mb-2 align-middle">
+            d-inline
+          </Text>
+        </div>
+        <Text appearance="subtle" className="mb-2 font-size-xs">
+          Elements with d-inline display as inline elements, appearing side by side and taking up only as much width as
+          necessary. They will wrap to new lines when they reach the container's edge.
+        </Text>
+        <div className="bg-secondary-lightest p-3 rounded-10 font-size-xs">
+          <pre className="m-0">
+            <code className="d-block">
+              {
+                '<Text appearance="default" className="d-inline bg-secondary-lightest border p-4 mr-2 mb-2 align-middle">d-inline</Text>'
+              }
+            </code>
+            <code className="d-block mt-2">
+              {
+                '<Text appearance="default" className="d-inline bg-secondary-lightest border p-4 mr-2 mb-2 align-middle">d-inline</Text>'
+              }
+            </code>
```

Comment:
Instead of using multiple `<code>` elements we can simply write all the text inside single `<code>`